src/tests/test_claim_types.py

```
@@ -0,0 +1,296 @@
+"""Tests for different claim types in the VeriFact pipeline."""
+
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.AsyncMock` imported but unused

Remove unused import

(F401)